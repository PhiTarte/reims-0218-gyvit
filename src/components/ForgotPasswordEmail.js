//TODO: create a email input
import React from "react"
import { Form, FormGroup, Label, Input } from "reactstrap"

export default class ForgotPasswordEmail extends React.Component {
  render() {
    return (
      <Form
        style={{
          "margin-bottom": "40px",
          "margin-top": "60px"
        }}
      >
        <FormGroup>
          <Label for="email">Email </Label>
          <Input type="email" name="email" id="IdEmail" placeholder="Email" />
        </FormGroup>
      </Form>
    )
  }
}
