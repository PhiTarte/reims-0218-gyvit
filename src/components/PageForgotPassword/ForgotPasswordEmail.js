//TODO: create a email input
import React from "react"
import { FormGroup, Label, Input } from "reactstrap"

export default class ForgotPasswordEmail extends React.Component {
  render() {
    return (
      <FormGroup
        style={{
          marginBottom: "40px",
          marginTop: "60px"
        }}
      >
        >
        <Label for="email">Email </Label>
        <Input type="email" name="email" id="IdEmail" placeholder="Email" />
      </FormGroup>
    )
  }
}
