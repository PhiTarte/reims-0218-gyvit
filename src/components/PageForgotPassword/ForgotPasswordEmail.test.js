//TODO CREATE TEST

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import ForgotPasswordEmail from "./ForgotPasswordEmail"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ForgotPasswordEmail />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<ForgotPasswordEmail />).toJSON()
  expect(tree).toMatchSnapshot()
})
