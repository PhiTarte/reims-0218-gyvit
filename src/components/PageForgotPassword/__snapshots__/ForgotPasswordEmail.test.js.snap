// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div
  className="form-group"
  style={
    Object {
      "margin-bottom": "40px",
      "margin-top": "60px",
    }
  }
>
  &gt;
  <label
    className=""
    htmlFor="email"
  >
    Email 
  </label>
  <input
    className="form-control"
    id="IdEmail"
    name="email"
    placeholder="Email"
    type="email"
  />
</div>
`;
